File: scr/HelloWord.php
<!DOCTYPE html>
<html>
<body>

    <?php
    // This is a simple Hello World script in PHP

    echo "Hello, World!";
    ?>

</body>
</html>

<!-- php -S localhost:8000 -->

<!-- Simple demo -->
<!DOCTYPE html>
<html>
<body>

<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $name = htmlspecialchars($_POST["name"]);
    echo "<h2>Hello, $name!</h2>";
}
?>

<h2>Enter your name:</h2>
<form method="post" action="<?php echo $_SERVER["PHP_SELF"]; ?>">
    <input type="text" name="name">
    <input type="submit" value="Submit">
</form>

</body>
</html>

File: scr/HelloWord.py
print('Hello World!')

# python HelloWord.py


# Simple demo
def add(a, b):
    return a + b

def subtract(a, b):
    return a - b

print("Select operation:")
print("1. Add")
print("2. Subtract")

choice = input("Enter choice (1/2): ")
num1 = float(input("Enter first number: "))
num2 = float(input("Enter second number: "))

if choice == '1':
    print(f"Result: {num1} + {num2} = {add(num1, num2)}")
elif choice == '2':
    print(f"Result: {num1} - {num2} = {subtract(num1, num2)}")
else:
    print("Invalid input.")
